specification/patient-care-aggregator-api.yaml

```diff
@@ -92,7 +91,7 @@ info:
     | ----------------- | ----------------------------------------------------------------------------- | ----------------------------- |
     | Sandbox           | Not available                                                                 | Not applicable                |
     | Integration test  | `https://int.api.service.nhs.uk/patient-care-aggregator-api/FHIR/R4/`         | NHS login - integration (AOS) |
```

Comment:
The URL for INT is https://int.api.service.nhs.uk/patient-care-aggregator-api/events

Reply:
Do you mean /events or do you mean /aggregator/events, as per prod?

Reply:
You're right, it's https://int.api.service.nhs.uk/patient-care-aggregator-api/aggregator/events

Reply:
Fixed